addons/website_mail/static/src/js/follow.js

```diff
@@ -131,12 +131,17 @@ publicWidget.registry.follow = publicWidget.Widget.extend({
                 });
                 return false;
             }
+
+            const turnstileCaptcha = $("input[name='turnstile_captcha']");
+            const turnstile = turnstileCaptcha[0] ? turnstileCaptcha[0].value : "";
```

Comment:
There is no dependence to `website_cf_turnstile` in `website_mail`, it's look weird to me to mention it.